src/Arcus.WebApi.Logging/ExceptionHandlingMiddleware.cs

```diff
@@ -62,20 +64,24 @@ public async Task Invoke(HttpContext context, ILoggerFactory loggerFactory)
             {
                 await _next(context);
             }
+            catch (BadHttpRequestException ex)
+            {
+                LogException(loggerFactory, ex);
```

Comment:
Let's add a comment here why we are catching these

Reply:
Ok